tests/istanbul/eip152_blake2/test_blake2.py

```diff
@@ -523,50 +555,412 @@ class ExpectedOutput(TestParameterGroup):
+                2: output.data_2,
+            }
+        )
+    }
+    state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.valid_from("Istanbul")
+@pytest.mark.parametrize(
+    ["data", "output"],
+    [
+        pytest.param(
+            b"",
+            ExpectedOutput(
+                call_succeeds=False,
+                data_1="0x00",
+                data_2="0x00",
+            ),
+            id="EIP-152-data0-large-gas-limit",
+        ),
+        pytest.param(
+            Blake2bInput(
+                rounds=12,
+                rounds_length=3,
+                h="48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b",
+                m="6162630000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+                t_0=3,
+                t_1=0,
+            ),
+            ExpectedOutput(
+                call_succeeds=False,
+                data_1="0x00",
+                data_2="0x00",
+            ),
+            id="EIP-152-data1-large-gas-limit",
+        ),
+        pytest.param(
+            Blake2bInput(
+                rounds=12,
+                rounds_length=5,
+                h="48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b",
+                m="6162630000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+                t_0=3,
+                t_1=0,
+            ),
+            ExpectedOutput(
+                call_succeeds=False,
+                data_1="0x00",
+                data_2="0x00",
+            ),
+            id="EIP-152-data2-large-gas-limit",
+        ),
+        pytest.param(
+            Blake2bInput(
+                rounds=12,
+                h="48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b",
+                m="6162630000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+                t_0=3,
+                t_1=0,
+                f=2,
+            ),
+            ExpectedOutput(
+                call_succeeds=False,
+                data_1="0x00",
+                data_2="0x00",
+            ),
+            id="EIP-152-data3-large-gas-limit",
+        ),
+        # pytest.param(
+        #     Blake2bInput(
+        #         rounds=8000000,
```

Comment:
This huge number of rounds unsurprisingly causes time outs. Is there another way to run this to keep this case covered?

Reply:
I tried running the test with a 5 minute timeout and it still timed out.

I'm thinking that lowering the number to something that doesn't time out with the current time limit should still provide sufficient coverage.

@gurukamath is this a known issue for this specific precompile?

Reply:
I lowered the rounds to `100_000`, updated the outputs, and added `marks=pytest.mark.slow()` to the test, and it finally fills 🎉 

I think what we can do is keep this test case as is and add `pytest.mark.skip("Times-out during fill")` or similar (which we should revisit at some point), and then add this alternate test case I described in the previous paragraph, what do you think?

Reply:
I will add the new cases and mark these with `skip`!